### ISSUE TITLE
[QUESTION] How to use ?

### ISSUE THREAD

**Issue by zowepsilon** [brython-dev/brython-pygame#2]
How to use this ?

**Comment by wkta** [comment]
Good question!

It took me a lot of time to figure out the answer by myself.

I have forked and patched the project... To make things **simpler** for everyone.
(a pull request is open since today to share my work)

In the meantime if you're curious you can check my forked repo

**Comment by PierreQuentel** [comment]
Bonjour Thomas,

L'auteur initial de pygame (Billy Earney) était un contributeur très actif de Brython il y a quelques années, et c'est pour cela que pygame est rattaché à brython-dev. Il a malheureusement abandonné le projet depuis longtemps et je ne pense pas qu'il réponde à ce message.

Je n'y ai pas participé et n'ai pas le temps de m'y mettre. Le mieux serait donc de fermer ce projet dans brython-dev et que tu reprennes le développement de pygame pour Brython dans ton repository.

Cordialement,
Pierre

(PS for non French speakers : the Google translate version is understandable ;-)

**Comment by wkta** [comment]
Bonjour,
entendu merci pour la réponse rapide ; je vais procéder ainsi. Au passage : bravo pour la qualité du travail réalisé sur Brython :100: . J'ai découvert l'outil il y a quelques jours j'en suis déjà fan.

**Comment by PierreQuentel** [comment]
Merci !
Je me suis inscrit pour suivre le développement de ta version. Bon courage !

**Comment by igordsm** [comment]
@wkta Are you open for collaborations on this? I might be able to contribute in the future and right now it is not possible to open issues in your fork. Maybe it would be even better to start a brand new project and just point to this one as the "old" version. 

Best regards.

**Comment by wkta** [comment]
@igordsm Yeah of course ! I would be very happy to collaborate! I have more or less solved the problem in a brand new project (private repo). So I have several pygame programs that run smoothly in the browser, it's great.… A few bugs remain however.

You know I have a lot to say but it's not the right place for this. Add me on Discord if you are curious: _Martialis hashtag ninety seventy six_

**Comment by wkta** [comment]
**Here is my proof-of-concept:**   [https://kata.games/demo/](https://kata.games/demo/)
Press arrow keys / your space bar to see for yourself that it works.

You can inspect the source-code of this page.
You will find it interesting, as always thanks to Brython. I think this issue can be closed / on peut fermer ce fil je pense.

If you plan to use pygame and share your creation on the Web, just contact me via Discord.
The toolkit I am creating isn't ready yet but it came into working state. Together we could improve it a lot more, then distribute new games online
